### ISSUE TITLE
bitfinex fetch_order error

### ISSUE THREAD

**Issue by noonnz** [ccxt/ccxt#1834]
It's respond 
"requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.bitfinex.com/v1/order/status"

- OS: MACOS
- Programming Language version: PYTHON
- CCXT version: lastest
- Exchange: bitfinex
- Method: fetch_order

**Comment by kroitor** [comment]
https://docs.bitfinex.com/v1/reference#rest-auth-order-status

Can you please show a minimal but a complete snippet of code to reproduce the problem and verbose output (add `({'verbose': True})` to contstructor params), as described here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

?

**Comment by kroitor** [comment]
https://github.com/bitfinexcom/bitfinex-api-node/issues/8

**Comment by kroitor** [comment]
Your version may not be the latest, please update ccxt to the most recent one as well.

```
mbp:python igorkroitor$ python
Python 2.7.13 (default, Jun  5 2017, 10:04:07)
[GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.42)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ccxt
b>>> b = ccxt.bitfinex({"apiKey": "MY_API_KEY","secret": "MY_SECRET"})
>>> b.enableRateLimit=True
>>> b.fetch_balance
<bound method bitfinex.fetch_balance of <ccxt.bitfinex.bitfinex object at 0x104562ad0>>
>>> b.fetch_balance()
{'info': [{u'available': u'0.00441364', u'currency': u'bch', u'amount': u'0.00441364', u'type': u'exchange'}, {u'available': u'0.0051689', u'currency': u'btc', u'amount': u'0.0051689', u'type': u'exchange'}, {u'available': u'0.0', u'currency': u'usd', u'amount': u'0.0', u'type': u'exchange'}], u'USD': {'total': 0.0, 'used': 0.0, 'free': 0.0}, u'BCH': {'total': 0.00441364, 'used': 0.0, 'free': 0.00441364}, 'free': {u'USD': 0.0, u'BCH': 0.00441364, u'BTC': 0.0051689}, 'used': {u'USD': 0.0, u'BCH': 0.0, u'BTC': 0.0}, u'BTC': {'total': 0.0051689, 'used': 0.0, 'free': 0.0051689}, 'total': {u'USD': 0.0, u'BCH': 0.00441364, u'BTC': 0.0051689}}
>>> b.fetch_order(123)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "ccxt/bitfinex.py", line 565, in fetch_order
    }, params))
  File "ccxt/base/exchange.py", line 310, in request
    return self.fetch2(path, api, method, params, headers, body)
  File "ccxt/base/exchange.py", line 307, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "ccxt/base/exchange.py", line 374, in fetch
    self.handle_errors(response.status_code, response.reason, url, method, None, self.last_http_response)
  File "ccxt/bitfinex.py", line 727, in handle_errors
    raise exact[message](feedback)
ccxt.base.errors.OrderNotFound: bitfinex {"message":"No such order found."}
>>>
```

 ↑ For a non-existent order you should get a message like that.

Can't help without seeing your code, let us know if this does not answer your question (paste code and verbose output).
